src/isomorphic/classic/element/__tests__/ReactElement-test.js

```diff
@@ -24,6 +24,10 @@ describe('ReactElement', function() {
   beforeEach(function() {
     require('mock-modules').dumpCache();
 
+    // Delete the native Symbol if we have one to ensure we test the
+    // unpolyfilled environment.
+    delete global.Symbol;
```

Comment:
We have evaluated requiring various ES6 polyfills but decided against it in the past. It was too much of a hassle for people and didn't work well with node especially.